src/client/serialize/SConscript

```diff
@@ -0,0 +1,31 @@
+"""Build FUSE client"""
+import daos_build
+
+def scons():
+    """Execute build"""
+    Import('env')
+
+    denv = env.Clone()
+
+    have_hdf5 = True
+
+    """Do hdf_serialize configure checks"""
+    if GetOption('help') or GetOption('clean'):
+        return denv
+    #If hdf5 installed build hdf_serialize library
+    conf = Configure(denv)
+    if not conf.CheckLibWithHeader('hdf5', 'hdf5.h', 'c'):
+        have_hdf5 = False
+    denv = conf.Finish()
```

Comment:
(pylint-assignment-from-no-return) Assigning result of a function call, where the function has no return